src/axom/inlet/Table.cpp

```diff
@@ -164,10 +173,6 @@ Table& Table::addGenericContainer(const std::string& name,
     table.m_sidreGroup->createViewScalar(detail::GENERIC_CONTAINER_FLAG,
                                          static_cast<int8>(1));
   }
-  else
```

Comment:
Should the warning stay if the field is required, or is that checked elsewhere?

Reply:
This is a good question - the `required` flag is checked in the verification phase, which is separate from the `add*` functions.  I've added some extra tests to ensure that required arrays cause verification to fail when they are empty.